### ISSUE TITLE
Find a support centre

### ISSUE THREAD

**Issue by rmccar** [ONSdigital/design-system#879]
Use this issue to discuss the [Find a support centre](https://ons-design-system.netlify.com/patterns/find-a-support-centre) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by trevorsaint** [comment]
### An important update that needs adding

For each link that goes to Google Maps we will need to override that text on the link with an aria-label.

I know for certain that this will fail on accessibility because all the user will hear is Find on Google repeated over and over again with no context for each result. We can fix this for screen reader users by adopting the following pattern.

**From this:**

`<a href="#0">Find on Google Maps</a>`

**To this:**

`<a href="#0" aria-label="Find University of Sheffield on Google Maps">Find on Google Maps</a>`

The aria label takes priority over what is read out by screen readers. 

**We could do one better for greater support:**

`<a href="#0">Find<span class="u-vh"> University of Sheffield </span>on Google Maps</a>`
